phpunit.xml.dist

```diff
@@ -69,7 +69,7 @@
         <env name="TESTS_ZEND_LDAP_USE_SSL" value="false" />
         <env name="TESTS_ZEND_LDAP_USERNAME" value="CN=someUser,DC=example,DC=com" />
         <env name="TESTS_ZEND_LDAP_PRINCIPAL_NAME" value="someUser@example.com" />
-        <env name="TESTS_ZEND_LDAP_PASSWORD" value="" />
+        <env name="TESTS_ZEND_LDAP_PASSWORD" value="insecure" />
```

Comment:
I changed this line as an empty password is not working at all for login. So it's a bad default